JavascriptAdapterTest errors when running with strict cookies on Firefox

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

adapter/javascript

### Describe the bug

When running the `-Pfirefox-strict-cookies` profile, `JavascriptAdapterTest` has a bunch of failures and errors, probably all related to the same root cause.

### Version

latest

### Expected behavior

Tests pass successfully

### Actual behavior

Sample error:

`[ERROR]   JavascriptAdapterTest.testSilentCheckSsoWithFallbackDisabled:199->AbstractJavascriptTest.assertOnLoginPage:170 » ConditionTimeout Assertion condition defined as a org.keycloak.testsuite.util.URLAssert URL expected to begin with: https://auth-server-127-0-0-1.nip.io:8543/auth/realms/test/protocol/openid-connect/auth ; actual URL: https://localhost:8543/auth/realms/test/protocol/openid-connect/auth?client_id=js-console&redirect_uri=https%3A%2F%2F127.0.0.1.sslip.io%3A8543%2Fauth%2Frealms%2Ftest%2Ftesting%2Fjavascript%2Findex.html&state=1c787b14-5031-4bec-8a79-7f40cdcfe9d2&response_mode=fragment&response_type=code&scope=openid&nonce=ba5991a8-f229-4a3a-aaf1-13fdf9fef8fb within 10 seconds.`

### How to Reproduce?

`mvn -f testsuite/integration-arquillian/tests/base/pom.xml clean install -Pauth-server-quarkus -Pfirefox-strict-cookies -Dauth.server.host=auth-server-127-0-0-1.nip.io -Dauth.server.host2=127.0.0.1.sslip.io -Dtest=JavascriptAdapterTest`

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/AbstractQuarkusDeployableContainer.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/AbstractQuarkusDeployableContainer.java
@@ -191,6 +191,10 @@ public abstract class AbstractQuarkusDeployableContainer implements DeployableCo
             commands.add("--log-level=" + System.getProperty("auth.server.quarkus.log-level"));
         }
 
+        if (System.getProperty("auth.server.host") != null) {
+            commands.add("-Dauth.server.host=" + System.getProperty("auth.server.host"));
+        }
+
         commands.addAll(getAdditionalBuildArgs());
 
         commands = configureArgs(commands);

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java
@@ -282,7 +282,7 @@ public class JavascriptAdapterTest extends AbstractJavascriptTest {
                 .init(iframeInterval)
                 .wait(2000, (driver1, output, events) -> { // iframe is initialized after ~1 second, 2 seconds is just to be sure
                     assertAdapterIsLoggedIn(driver1, output, events);
-                    final String logMsg = "3rd party cookies aren't supported by this browser.";
+                    final String logMsg = "Your browser is blocking access to 3rd-party cookies, this means:";
                     if (SuiteContext.BROWSER_STRICT_COOKIES) {
                         // this is here not really to test the log but also to make sure the browser is configured properly
                         // and cookies were blocked

